ClientApp/src/Pages/GettingStarted/LocationData.json

```diff
@@ -0,0 +1,34 @@
+
+
+{ 
```

Comment:
Could we make this a typescript file rather than a JSON one?
That way we get all the nice type safety :) 

ie just
```ts
interface LocationData {
    name: string;
    imageSrc: string;
    description: string;
    // ...
}

export const scotland: LocationData = {
    name: "Gairloch, Wester Ross",
    // ...
}